sphinx/source/docs/user_guide/interaction.rst

```diff
@@ -290,3 +290,26 @@ interactions such as a box zoom, wheel scroll or pan.
 .. |figure| replace:: :func:`~bokeh.plotting.figure`
 
 .. |bokeh.plotting| replace:: :ref:`bokeh.plotting <bokeh.plotting>`
+
+.. _userguide_interaction_actions_in_python:
+
+CustomJS with a Python function
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A CustomJS callback can also be implemented as a Python function, which
+is then transpiled to JavaScript using PyScript. This makes it easier
```

Comment:
"transpiled" is a very techy term. Maybe just "translated" or even "turned into"